mappings/net/minecraft/client/render/entity/EntityRenderer.mapping

```diff
@@ -1,6 +1,7 @@
 CLASS net/minecraft/class_1136 net/minecraft/client/render/entity/EntityRenderer
 	FIELD field_10640 dispatcher Lnet/minecraft/class_1135;
 	FIELD field_10641 blockRenderer Lnet/minecraft/class_2336;
+	FIELD field_10642 getAlwaysFalse Z
```

Comment:
Looks like the field and method names were switched? Also, the name seems a bit odd. If some code is unused or seems to have no purpose, it's likely a relic from older versions in which case it's best left unmapped.